src/styles/blocks/star.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  width: 96px;
+  margin-left: 18px;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(-n + 4) {
+    margin-right: 0;
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
You have set margin-right to 0 for active stars. This will cause the stars to stick together without any space in between, which is not visually appealing. You should remove this line.